### ISSUE TITLE
Monster Hunter Wilds Beta exits on launch

### ISSUE THREAD

**Issue by notjl** [Whisky-App/Whisky#1192]
### Description

Monster Hunter Wilds Beta Test via Steam doesn't run. When you clicked `Play` on Steam, it briefly shows `Running install script ()` then closes. It doesn't put you in any game screen or anything. I tried running it with DXVK on and off, still doesn't change anything.

### Steps to reproduce

1. Launch Steam
2. Launch Monster Hunter Wilds Beta
3. Game exits

### Expected behaviour

I expect it to load the game and show the main menu telling me that the Beta test is only for November 1-4.

### Logs

Whisky Version: 2.3.3
Date: 2024-10-31T08:11:23Z
macOS Version: 15.0.1

Bottle Name: Steam
Bottle URL: /Users/------------------/Library/Containers/com.isaacmarovitz.Whisky/Bottles/CA5ABDA3-2E7C-4DED-81CD-88086D905CAC

Wine Version: 7.7.0
Windows Version: win11
Enhanced Sync: msync

Metal HUD: false
Metal Trace: false

DXVK: true
DXVK Async: true
DXVK HUD: off

Arguments: start /unix /Users/----------------/Library/Containers/com.isaacmarovitz.Whisky/Bottles/CA5ABDA3-2E7C-4DED-81CD-88086D905CAC/drive_c/Program Files (x86)/Steam/steamapps/common/MonsterHunterWildsBetatest/MonsterHunterWildsBeta.exe

Environment:
{
    "DXVK_ASYNC" = 1;
    "GST_DEBUG" = 1;
    WINEDEBUG = "fixme-all";
    WINEDLLOVERRIDES = "dxgi,d3d9,d3d10core,d3d11=n,b";
    WINEESYNC = 1;
    WINEMSYNC = 1;
    WINEPREFIX = "/Users/-----------------/Library/Containers/com.isaacmarovitz.Whisky/Bottles/CA5ABDA3-2E7C-4DED-81CD-88086D905CAC";
}

06e8:err:winediag:ntlm_check_version ntlm_auth was not found or is outdated. Make sure that ntlm_auth >= 3.0.25 is in your path. Usually, you can find it in the winbind package of your distribution.

### What version of Whisky are you using?

2.3.2

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [x] Yes my issue is written in English

**Comment by Narukane** [comment]
I’m facing the same issue. I also tried running it with Crossover (version 24.0.5), but the result is still the same. However, when using Crossover, there’s a crash report from MH Wilds.
![Image](https://github.com/user-attachments/assets/745c7cad-6ac8-4e3a-9ce9-f59abc11bb0e)

**Comment by ElCrucible** [comment]
yes i also got the same exact issue, even cross checked wether monster hunter world runs properly, and MHWI runs very well so, it should be the data from the monster hunter wilds beta test
